tailwind.config.ts

```diff
@@ -2,7 +2,18 @@
 module.exports = {
   content: ["./src/**/*.{ts,tsx}"],
   theme: {
-    extend: {},
+    extend: {
+      fontSize: {
+        description: ["25.26px", "35.98px"],
+      },
+      lineHeight: {
+        185: "185px",
+        87: "87px",
+      },
```

Comment:
let's not hardcode in values for font size and line height.  we'll go more in depth into this tomorrow but if you don't want to wait til tomorrow try experimenting with vh vw, % units.